### ISSUE TITLE
ProhibitUnusedConstant is confused by ternary expression

### ISSUE THREAD

**Issue by petdance** [gugod/Perl-Critic-TooMuchCode#5]
```
$ cat foo.pl
#!/usr/bin/perl

use constant FOO => 1;
use constant BAR => 2;

$x = $y ? FOO : BAR;
```

```
$ perlcritic -s TooMuchCode::ProhibitUnusedConstant  foo.pl
foo.pl: Unused constant.
    3:use constant FOO => 1;
```

`FOO` is indeed used, but the policy complains anyway.

**Comment by gugod** [comment]
Fixed in the 0.11 release.

**Comment by gugod** [comment]
@petdance thanks for the report 👍 I needed to implemented a workaround of a [PPI-misparse]. I'm not sure if that covers enough edge cases... but we'll see.

[PPI-misparse]: https://github.com/adamkennedy/PPI/issues/62
